std_msgs/String robot_name
---
sorting_robot/Pickup pickup